### PR TITLE
Refinery Update - Color Fixers

### PR DESCRIPTION
**Pull request by Kitsunemitsu**
## New Weapon Sets
 - Added Leaflet, an office with durability and sky-high damage
 - Added Liu Association Section 1 and 2. Swords, spears and gloves that gain 10% damage for every human around them
 
## New Weapons
 - Added Dong-hwan's weapon, a unique weapon that has multiple different functions and attacks
 - Added Index Spear
 - Added Mini Zwei sword
 - Added new Fullstop gat
 - Added in the Shi assassin Knife, a weapon that instantly kills you if you hit yourself
 - Added Yun's Glove
 - Fills out the Shi BOD weapons
 - Added Vermillion Cross
 - Added Resonant Scythe, by Yurai [Post-poned for quality]
 - Added Kali's Mimicry
 

## New Armor
 - Added Ncorp armor
 - Added Blue Reverb Armor
 - Added new Rosespanner assassin jacket
 - Added Liu Section 2 Armor 
 - Added Liu Section 1 Armor (full set of 3)
 - Added Liu Section 5 Armor (full set of 3)

## New Banners
 - Added Liu Banner

### Fixes
 - Index weapons no longer scale infinitely, get better scaling

## Changelog
:cl:
add: Refinery Additions
fix: fixed issues with Index weapons scaling infinitely
/:cl:

### CODE REVIEW

File: code/game/objects/items/ego_weapons/non_abnormality/shi.dm [hunk 1]
```diff
@@ -63,3 +93,28 @@
...
+	attribute_requirements = list(
+							FORTITUDE_ATTRIBUTE = 80,
+							PRUDENCE_ATTRIBUTE = 80,
+							TEMPERANCE_ATTRIBUTE = 100,
+							JUSTICE_ATTRIBUTE = 60
+							)
+
+//Grade 3
+/obj/item/ego_weapon/city/shi_assassin/director
+	name = "shi association veteran sheathed blade"
```
[the remark this answers is not in the document]
Comment (Kitsunemitsu) [reply code/game/objects/items/ego_weapons/non_abnormality/shi.dm:111]: Yeah, I missed that one. This PR is still a little WIP as I have liu stuff incoming

File: code/game/objects/items/ego_weapons/non_abnormality/shi.dm [hunk 2]
```diff
@@ -1,14 +1,44 @@
...
+	..()
+	force = force_update
+	if(!CanUseEgo(user))
+		return
+	if(target == user)
+		force = 999999
+		for(var/mob/M in GLOB.player_list)
+			to_chat(M, "<span class='userdanger'>[uppertext(user.real_name)] has gone out with honor. 灰から灰へ</span>")
+		new /obj/effect/temp_visual/BoD(get_turf(target))
+		force_update *= 1.2
```
Comment (dsmith328) [inline code/game/objects/items/ego_weapons/non_abnormality/shi.dm:33]: As written, this hits the user for 44 red damage, then if the user is the target sets the force to 999999. This means they could stab someone with that.

**Review by dsmith328 — CHANGES_REQUESTED** [review]
Overall, I think it'd be infinitely cooler if the Blue Reverb scythe had more relevant stuff to the Blue Reverb instead of just being Justitia+

File: code/game/objects/items/ego_weapons/non_abnormality/color_fixer.dm [hunk 3]
```diff
@@ -0,0 +1,126 @@
+//Color fixer stuff is all Grade 1
+/obj/item/ego_weapon/city/vermillion
+	name = "Vermillion Cross"
+	desc = "Ashes to ashes, dust to dust."
+	special = "Use in hand to cut HP in half. Each kill for the next 15 seconds increases damage by 1.5x, and heals you 10%."
+	icon_state = "vermillion"
+	inhand_icon_state = "vermillion"
```
[the remark this answers is not in the document]
Comment (Kitsunemitsu) [reply code/game/objects/items/ego_weapons/non_abnormality/color_fixer.dm:7]: It's for later..... standard practice.

File: ModularTegustation/Teguicons/lc13_weapons.dmi
[the remark this answers is not in the document]
Comment (Kitsunemitsu) [reply ModularTegustation/Teguicons/lc13_weapons.dmi:1]: Soon, the mimicry from Nothing there will be a scythe. Voted by the community and is in #1090

File: code/game/objects/items/ego_weapons/non_abnormality/color_fixer.dm [hunk 4]
```diff
@@ -0,0 +1,126 @@
...
+/obj/item/ego_weapon/city/resonantscythe/attack(mob/living/M, mob/living/user) //Once more, but using Justitia's code. It'd be a surprise if this actually worked.
+	if(!CanUseEgo(user))
+		return
+	if(world.time > combo_time)
+		combo = 0
+	combo_time = world.time + combo_wait
+	switch(combo)
+		if(5)
+//			hitsound = 'sound/weapons/fixer/Blue_Argalria_Far_Atk1.ogg'
+			force *= 1.0
```
[the remark this answers is not in the document]
Comment (Kitsunemitsu) [reply code/game/objects/items/ego_weapons/non_abnormality/color_fixer.dm]: I didnt' code this

File: code/game/objects/items/ego_weapons/non_abnormality/shi.dm (hunk 2 above)
Comment (Kitsunemitsu) [reply code/game/objects/items/ego_weapons/non_abnormality/shi.dm]: Sorry, I put the ..() in the wrong place. Fixed.

File: code/game/objects/items/ego_weapons/non_abnormality/zwei.dm [hunk 5]
```diff
@@ -95,6 +94,14 @@
...
 							)
 
+//Mini Zwei
+/obj/item/ego_weapon/city/zweihander/knife
+	name = "mini zweihander"
+	desc = "A handheld, one handed Zweihander."
+	icon_state = "zwei_mini"
+	force = 32
+	attack_speed = 1
+
```
[the remark this answers is not in the document]
Comment (Kitsunemitsu) [reply code/game/objects/items/ego_weapons/non_abnormality/zwei.dm:104]: Great point, main idea was to get a zweihander that doesn't take 13 Years to swing.

File: code/modules/clothing/suits/ego_gear/realized.dm [hunk 6]
```diff
@@ -226,6 +226,12 @@
 	armor = list(RED_DAMAGE = 80, WHITE_DAMAGE = 50, BLACK_DAMAGE = 70, PALE_DAMAGE = 60)
 	flags_inv = HIDEJUMPSUIT|HIDEGLOVES|HIDESHOES
 
+/obj/item/clothing/suit/armor/ego_gear/realization/bell_tolls
```
[the remark this answers is not in the document]
Comment (Kitsunemitsu) [reply code/modules/clothing/suits/ego_gear/realized.dm:229]: Kinda? Basically playerbase voted for smth, I did half of it and I need to hold up the other half of the promise.

File: code/game/objects/items/ego_weapons/non_abnormality/color_fixer.dm [hunk 7]
```diff
@@ -0,0 +1,126 @@
...
+	addtimer(CALLBACK(src, .proc/Return, user), 15 SECONDS)
+
+
+/obj/item/ego_weapon/city/vermillion/attack(mob/living/target, mob/living/carbon/human/user)
+	..()
+	if(force != initial(force))
+		to_chat(user, "<span class='userdanger'>ANOTHER.</span>")
+		force*=1.5
+		user.adjustBruteLoss(-user.maxHealth*0.1)
+		force = initial(force)
```
[the remark this answers is not in the document]
Comment (Kitsunemitsu) [reply code/game/objects/items/ego_weapons/non_abnormality/color_fixer.dm:42]: God IDK what I was doing here, one sec.

File: code/modules/clothing/suits/ego_gear/realized.dm (hunk 6 above)
[the remark this answers is not in the document]
Comment (dsmith328) [reply code/modules/clothing/suits/ego_gear/realized.dm:229]: Nope, just making sure it's SUPPOSED to be in this PR

**Review by dsmith328 — APPROVED** [review]

**Review by LanceSmites328 — APPROVED** [review]

**Pull request merged**